src/plugins/translate/plugin.translate.js

```diff
@@ -99,20 +100,21 @@ export class TranslatePlugin extends BookReaderPlugin {
     });
     
     for (const page of visiblePageContainers) {
-      this.translateRenderedLayer(page);
+      this.translateRenderedLayer(page, 0);
     }
     for (const loadingPage of hiddenPageContainers) {
-      this.translateRenderedLayer(loadingPage);
+      this.translateRenderedLayer(loadingPage, 1000);
     }
   }
 
   /** @param {HTMLElement} page */
-  async translateRenderedLayer(page) {
+  async translateRenderedLayer(page, priority) {
     if (this.br.mode == this.br.constModeThumb) {
       return;
     }
 
     const pageIndex = page.dataset.index;
+    const pagePriority = parseInt(pageIndex) + priority;
```

Comment:
There's a bit of a gotcha with `parseInt` in js! It by default will not necessarily parse the string as base 10 😁 If the string begins with a "0x" (not just 0 as I said earlier!) it is parsed as base 16. So for best practice, we generally either always use the radix parameter, or we just use parseFloat (my preference) since it is quick and reliable/consistent.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#description
```suggestion
    const pagePriority = parseFloat(pageIndex) + priority;
```